index.html

```diff
@@ -398,5 +398,66 @@ <h2>Transferable MediaStreamTrack</h2>
       In a sense, between these steps, the data holder is attached to the underlying source as if it was a track.</p>
     </div>
   </section>
+  <section>
+    <h2>VoiceIsolation constraint</h2>
+    <div>
+      <p>Some platforms offer functionality for voice isolation:
+      Attempting to remove all parts of an audio track that do not
+      correspond to a human voice. Some platforms even attempt to
+      remove extraneous voices, leaving the "main voice" as the
+        dominant component of the audio. The exact methods used may
+      vary between implementations.
+      </p>
+      <p>
+        This will have large effects on audio that is presented for
+        other reasons than to transmit voice (for instance music or
+        ambient noises), so needs to be off by default.
+      </p>
+      <p>
+        This constraint is a stronger version of noise cancellation,
+        which means that if the "noiseSuppression" constraint is set
+        to false and "voiceIsolation" is set to true, the value of
+        "noiseCancellation" will be ignored.
+      </p>
+      <p>
+        This constraint has no such relationship with any other
+        constraint; in particular it does not affect echoCancellation.
```

Comment:
With echo cancellation + voice isolation, the combinations are:

(false, false): not a problem
(true, false): not a problem if echo cancellation is supported at all
(false, true): this means that the app has decided that echo won̈́'t be a problem, so shouldn't be a problem to support
That leaves (true, true) as the only possibly conflicted case.

Since this is likely to be the most commonly used combination for videoconferencing, I would think that it would actually make more sense to say that all combinations of voiceIsolation and echoCancellation MUST be supported. but since it's a "best effort" constraint, it is perfectly reasonable to do a bad job of voice isolation if you can't do better.